central/detection/lifecycle/manager.go

```diff
@@ -22,6 +23,7 @@ import (
 const (
 	rateLimitDuration            = 10 * time.Second
 	indicatorFlushTickerDuration = 1 * time.Minute
+	baselineFlushTickerDuration  = 5 * time.Second
```

Comment:
Since the observation window is an hour, most the time when we process the deployments we do nothing so somewhere up above Gorman suggested lowering the interval.